### DISCUSSION TITLE
Is there an example of a working PXE manifest ?

### DISCUSSION THREAD

**Discussion by SR-G** [DSpeichert/netbootd#13] [Q&A]
Hello,
I'm really new to PXE / iPXE and, to be honest, it's really difficult to have something running (many options, many configurations, ...).
Hopefully i found netbood, and it works perfectly for my (recent) hardware with iPXE (the "everything packaged in the same binary" approach is really a breath of air in that complex ecosystem).

However, i have an old board (a miniforum NUC "UM250") only allowing "PXE" and not "iPXE" - and here, i'm not able to find a working configuration - so, does anyone have a netbood manifest showing what should be configured for "PXE" only ?

(side note : i'm willing to use flatcar linux after boot, in case it would be relevant)

Thanks in advance and regards.

**Comment by DSpeichert** [comment]
Technically, all of it is "just PXE" because we first use it to boot iPXE which then does a second PXE sequence. It should work on anything that does PXE and runs architecture that iPXE runs on.

You can disable iPXE [here](https://github.com/DSpeichert/netbootd/blob/8e91cff1e3a8375f371d4059132250ffe1ce2104/examples/ubuntu-1804.yml#L32) and then change bootFilename to something else, which becomes a replacement for iPXE binary (see logic [here](https://github.com/DSpeichert/netbootd/blob/8e91cff1e3a8375f371d4059132250ffe1ce2104/dhcpd/handler.go#L136-L152))

**Comment by SR-G** [comment]
Thanks for your answer.

I'm still unable to know exactly what to do (after a few tryes, i still haven't been able to use "pure PXE" with this old mini-board).

But ... at second thought, i'm now going for a (maybe) simpler solution in the end : booting this device from a USB key in which the ipxe.iso image has been flashed, and then everything is of course working fine (so i won't have to maintain something purely PXE related for one device).

Thanks again for your netbootd nicely integrated stack, and i hope the project is not abandonned ;)

**Reply by DSpeichert** [reply]
"Pure PXE" would be DHCP and TFTP, whereas iPXE adds HTTP and more. You'd need to boot something instead of iPXE that you can use to launch your installer/system/whatever - grub or pxelinux/syslinux. Glad you found a workaround!
